### ISSUE TITLE
slot machine "skin you alive" quote guaranteed to appear over "please bet at least 1 BB"

### ISSUE THREAD

**Issue by ieatcrayons3** [Bobbyperson/nocaro#26]
lines 3214 to 3221 
```python
 if amount <= 0:
            await ctx.reply("I will skin you alive.")
            return
        if await econ.get_bal(ctx.author) < amount:
            await ctx.reply("Not enough money!!! go `,map`")
            return
        if amount < 1:
            await ctx.send("Please bet at least 1 bouge buck.")
            return
``` 

are guaranteed to return "I will skin you alive." if money input isn't positive int. not sure if this is intentional? but "please bet at least 1BB" will never run

**Comment by Bobbyperson** [comment]
so this command is one of the more poorly written parts of code, and really needs a rewrite probably using #13 
even for an account over 10 days old it returns with the error `discord.ext.commands.errors.CommandInvokeError: Command raised an exception: TypeError: unhashable type: 'numpy.ndarray'` likely due to one of the dependencies updating since that was never properly tracked until somewhat recently

for now i'll make a commit to disable the history command but leave this issue open to track the progress for the rewrite

**Comment by Bobbyperson** [comment]
fixed with 695bf6ddaa15cc7666ffac97f6919762d935c88b, thanks
